src/sql/hydra.sql

```diff
@@ -84,9 +84,13 @@ create table Jobsets (
     startTime     integer, -- if jobset is currently running
     type          integer not null default 0, -- 0 == legacy, 1 == flake
     flake         text,
-    check (schedulingShares > 0),
-    check ((type = 0) = (nixExprInput is not null and nixExprPath is not null)),
-    check ((type = 1) = (flake is not null)),
+    constraint jobsets_schedulingshares_nonzero_check check (schedulingShares > 1),
```

Comment:
Should this be `schedulingShares > 0`?

Reply:
I made a copy-paste :thinking: good catch!

Reply:
Fixed thanks!